tests/console/clamav.pm

```diff
@@ -42,8 +42,11 @@ sub scan_and_parse {
 sub run {
     select_serial_terminal;
 
-    zypper_call('in clamav vim');
+    zypper_call('in clamav');
     zypper_call('info clamav');
+    # Create a random file to replace /usr/bin/vim
```

Comment:
What for?

Reply:
In the ticket it says "or create a random file replacing vim"